src/components/Loader/Loader.tsx

```diff
@@ -1,8 +1,13 @@
-import React from 'react';
+import React, { memo } from 'react';
+import { wait } from '../../api';
 import './Loader.scss';
 
-export const Loader: React.FC = () => (
-  <div className="Loader" data-cy="loader">
-    <div className="Loader__content" />
-  </div>
-);
+export const Loader: React.FC = memo(() => {
+  wait(300);
```

Comment:
it doesnt work
Loader - its component - its sync function
next code below won't wait while this promise will resolve

just undo it

dont modify Loader component and dont use wait function outside api.ts file